[2023-06-23 20:05:27,966] INFO _log() L224   * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.15.96:5000 (Press CTRL+C to quit) - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-06-23 20:05:27,967] INFO _log() L224   * Restarting with stat - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-06-23 20:05:32,013] INFO _log() L224  127.0.0.1 - - [23/Jun/2023 20:05:32] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-06-23 20:05:32,040] INFO _log() L224  127.0.0.1 - - [23/Jun/2023 20:05:32] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-06-23 20:05:32,041] INFO _log() L224  127.0.0.1 - - [23/Jun/2023 20:05:32] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-06-23 20:05:32,041] INFO _log() L224  127.0.0.1 - - [23/Jun/2023 20:05:32] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-06-23 20:05:32,339] INFO _log() L224  127.0.0.1 - - [23/Jun/2023 20:05:32] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-06-23 20:05:32,347] INFO _log() L224  127.0.0.1 - - [23/Jun/2023 20:05:32] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-06-23 20:05:43,070] ERROR log_exception() L1449 Exception on /tecnologia [GET] - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/flask/app.py L1449
Traceback (most recent call last):
  File "/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/flask/app.py", line 1519, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/flask/app.py", line 1517, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/flask/app.py", line 1503, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/flask_openapi3/scaffold.py", line 109, in wrapper
    resp = _do_wrapper(
           ^^^^^^^^^^^^
  File "/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/flask_openapi3/do_wrapper.py", line 135, in _do_wrapper
    response = func(**request_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/app.py", line 287, in get_tecnologia
    logger.debug(f"Aplica√ß√£o encontrada: '{tecnologia.nome}'")
                                           ^^^^^^^^^^^^^^^
AttributeError: 'Tecnologia' object has no attribute 'nome'
[2023-06-23 20:05:43,081] INFO _log() L224  127.0.0.1 - - [23/Jun/2023 20:05:43] "[35m[1mGET /tecnologia?id=1 HTTP/1.1[0m" 500 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-06-23 20:09:13,999] INFO _log() L224   * Detected change in '/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/app.py', reloading - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-06-23 20:09:14,124] INFO _log() L224   * Restarting with stat - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-06-23 20:09:18,434] INFO _log() L224  127.0.0.1 - - [23/Jun/2023 20:09:18] "GET /tecnologia?id=1 HTTP/1.1" 200 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-06-23 20:09:48,252] INFO _log() L224  127.0.0.1 - - [23/Jun/2023 20:09:48] "GET /tecnologia?id=2 HTTP/1.1" 200 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-06-23 20:10:02,155] WARNING get_tecnologia() L284  Erro ao buscar a aplica√ß√£o '0', Tecnologia n√£o encontrada na base :/ - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/app.py L284 
[2023-06-23 20:10:02,156] INFO _log() L224  127.0.0.1 - - [23/Jun/2023 20:10:02] "[33mGET /tecnologia?descricao=Java HTTP/1.1[0m" 404 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-06-23 20:10:15,894] INFO _log() L224  127.0.0.1 - - [23/Jun/2023 20:10:15] "GET /tecnologia?descricao=JAVA HTTP/1.1" 200 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-06-23 20:15:47,334] INFO _log() L224  127.0.0.1 - - [23/Jun/2023 20:15:47] "POST /tecnologia HTTP/1.1" 200 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-06-23 20:16:14,279] WARNING get_tecnologia() L284  Erro ao buscar a aplica√ß√£o '0', Tecnologia n√£o encontrada na base :/ - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/app.py L284 
[2023-06-23 20:16:14,280] INFO _log() L224  127.0.0.1 - - [23/Jun/2023 20:16:14] "[33mGET /tecnologia?descricao=Banco%20de%20Dados HTTP/1.1[0m" 404 - - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-06-23 20:20:30,573] INFO _log() L224   * Detected change in '/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/schemas/tecnologia.py', reloading - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-06-23 20:20:30,699] INFO _log() L224   * Restarting with stat - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-06-23 20:22:55,090] INFO _log() L224   * Detected change in '/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/app.py', reloading - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-06-23 20:22:55,212] INFO _log() L224   * Restarting with stat - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-06-23 20:31:07,172] INFO _log() L224   * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.15.96:5000 (Press CTRL+C to quit) - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
[2023-06-23 20:31:07,173] INFO _log() L224   * Restarting with stat - call_trace=/home/leandro/Documentos/Treinamentos/P√≥s Engenharia Software/MVP1/myProjects/app_api/env/lib/python3.11/site-packages/werkzeug/_internal.py L224 
